lib/msal-browser/docs/logout.md

```diff
@@ -49,7 +49,7 @@ The logout API also accepts an object for different configuration options:
 ## Code Snippet
 
 ```javascript
-const currentAccount = getAccountByUsername(username);
+const currentAccount = getAccountById(homeAccountId);
```

Comment:
Should this be something like `msalInstance.getAccountbyId`?

Reply:
Good catch! Fixed.